src/JSTest.TestAdapter/TestExecutor.cs

```diff
@@ -71,6 +77,12 @@ public void RunTests(IEnumerable<string> sources, IRunContext runContext, IFrame
             catch (JSTestException e)
             {
                 frameworkHandle.SendMessage(TestMessageLevel.Error, e.ToString());
+

```

Comment:
Same here,
no need to specifically check
+ log with `EqtTrace.Error`

Reply:
Done.